app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 3000
+
+const routes = require('./routes/')
+
+app.use(express.json())
+app.use(express.urlencoded({extended:true}))
+
+app.use('/',routes )
```

Comment:
Indentasinya dirapihkan

```suggestion
app.use('/', routes)
```